plugin/src/main/cpp/export/pico_export_plugin.cpp

```diff
@@ -200,5 +222,34 @@ String PicoEditorExportPlugin::_get_android_manifest_application_element_content
 		contents += "        <meta-data tools:node=\"replace\" android:name=\"Hand_Tracking_HighFrequency\" android:value=\"1\" />\n";
 	}
 
+	HybridType hybrid_type = _get_hybrid_app_option_value();
+	if (hybrid_type != HYBRID_TYPE_DISABLED) {
+		contents += "        <meta-data tools:node=\"replace\" android:name=\"pvr.app.type\" android:value=\"2d\" />\n";
+
+		contents +=
+				"        <activity android:name=\"org.godotengine.openxr.vendors.GodotPanelApp\" "
+				"android:process=\":GodotPanelApp\" "
+				"android:label=\"@string/godot_project_name_string\" "
+				"android:theme=\"@style/GodotAppSplashTheme\" "
+				"android:launchMode=\"singleInstancePerTask\" "
+				"android:excludeFromRecents=\"false\" "
+				"android:exported=\"true\" "
+				"android:screenOrientation=\"landscape\" "
+				"android:configChanges=\"orientation|keyboardHidden|screenSize|smallestScreenSize|density|keyboard|navigation|screenLayout|uiMode\" "
+				"android:resizeableActivity=\"true\" "
+				"tools:ignore=\"UnusedAttribute\" >\n"
+				"          <intent-filter>\n"
+				"            <action android:name=\"android.intent.action.MAIN\" />\n"
+				"            <category android:name=\"android.intent.category.DEFAULT\" />\n";
+
+		if (hybrid_type == HYBRID_TYPE_START_AS_PANEL) {
+			contents += "            <category android:name=\"android.intent.category.LAUNCHER\" />\n";
+		}
```

Comment:
We should try to move this common logic into the base class, and provide overrides to add the platform specific bits.

Reply:
This one is a bit tricky to centralize, because different platform could have special things that need to put anywhere in this XML.

Between Meta and Pico, though, we could probably merge them fairly easily - the only difference is that Meta needs an additional `<category>`, and Pico needs an additional `<meta-data>` (which doesn't need to be included with this block, it can be put before or after so that's fine). So, if we had a function that could generate this with optional stuff to put in `<intent-filter>` then it could be shared in this case.

Reply:
I'd expect the `<activity android:name="org.godotengine.openxr.vendors.GodotPanelApp" ...>` tag to remain the same between the different platforms, so we could just add a helper function to generate that tag and return it for insertion here.
Since we need to add logic to customize the activity attributes based on export options, we can consolidate that logic in that helper method.

Reply:
Ok! I've done that in my latest push